financial-templates-lib/clients/ExpiringMultiPartyClient.js

```diff
@@ -90,32 +93,49 @@ class ExpiringMultiPartyClient {
     return this.lastUpdateTimestamp;
   }
 
-  async update() {
-    // Since this function can have a relatively long run-time, we should be safe and check contract state explicitly at the same block height.
-    // This way we won't experience errors where the contract state changes between calls.
-    this.latestBlock = await this.web3.eth.getBlockNumber();
+  // Returns web3.eth.Contract with an ERC20 token interface at
+  // the tokenCurrency address
+  async getTokenCurrency() {
+    const tokenCurrencyAddress = await this.emp.methods.tokenCurrency().call({}, this.latestBlock);
```

Comment:
I think you need to pull and merge with the latest `master` code, which no longer passes a `this.latestBlock` into `.call()`'s on contracts. The issue caused by `latestBlock` is explained [here](https://github.com/UMAprotocol/protocol/issues/1811)

and the associated PR is [here](https://github.com/UMAprotocol/protocol/pull/1805)